packages/cfpb-layout/src/molecules/heroes.less

```diff
@@ -164,4 +164,69 @@
             }
         } );
     }
+
+    &__jumbo {
+        background-color: @white;
+
+        .m-hero_subhead {
+            .lead-paragraph();
+        }
+
+        .respond-to-max( @bp-xs-max, {
+            .m-hero_wrapper {
+                // Remove wrapper bottom padding.
+                padding-bottom: 0;
+            }
+
+            .m-hero_image-wrapper {
+                position: relative;
+                border-width: 0;
+            }
+        } );
+
+        .respond-to-min( @bp-sm-min, {
+            .m-hero_wrapper {
+                // Position the hero image all the way to the right plus a little
+                // extra so that it flows into the gutter
+                background-position: calc( 100% + @grid_gutter-width ) center;
+                background-repeat: no-repeat;
+                background-size: 50%;
+                border-right: 1px solid @gray-40;
+                border-bottom: 1px solid @gray-40;
+                border-left: 1px solid @gray-40;
```

Comment:
Alternatively, it looks like the above three lines could be written as two lines:
```
border: 1px solid @gray-40;
border-top: none;
```